public/components/page-dashboard/payouts/payouts.stache

```diff
@@ -24,10 +24,10 @@
 							<span>{{ formatDollarAmount(scope.vm.contributionMonths.getTotalForAllPayoutsForContributor(contributorRef, scope.vm.contributionMonth)) }}</span>
 						</td>
 						{{# each(scope.vm.contributionMonth.sortedMonthlyOSProjects) }}
-							{{# scope.vm.payoutFor(../contributorRef, osProjectRef) }}
+							{{# with( scope.vm.payoutFor(../contributorRef, osProjectRef) ) }}
```

Comment:
This change isn't necessary.  I just like the clarity of what is happening that `with` provides.